### ISSUE TITLE
Illegal image formats in resulting GLTF file.

### ISSUE THREAD

**Issue by vsantosu** [pissang/claygl#54]
When converting a FBX to a GLTF file, either binary or JSON gltf, the script(fbx2gltf.py) should do image transformation to PNG or JPEG in order to support most rendering engines. To reproduce the bug, try to transform a FBX with reference to a TGA image.

**Comment by pissang** [comment]
Yes it will be useful. Didn't do it because it needs other image processing libraries like `Pillow`
